reference/docs-conceptual/dev-cross-plat/create-feedback-provider.md

```diff
@@ -0,0 +1,682 @@
+---
```

Comment:
The resolution of `-arch.png` and `-output.png` seems to be not good. Any chance to make them clearer?